src/classes/8.rst

```diff
@@ -14,12 +14,55 @@ Career Module
 
 -  Understand how to continue learning on your own
 
-
 Prep Work
 ---------
 
+Tasks
+^^^^^
+
+- Read `Being a Developer After 40 <https://deprogrammaticaipsum.com/2016/04/25/being-a-developer-after-40/>`_. Read points 4 and 5 under the Advice for the Young at Heart section. The entire article is a good read, but the author specifically mentions tips about continued education in points 4 and 5.
+- Read `4 Things I Wish I Would Have Known When I Started My Software Development Career <https://simpleprogrammer.com/software-development-career/>`_. This brief article in which a developer shares his thoughts on programming fads, and continued learning.
+- Review week 8 of the demo projects: :ref:`Event List Week 8 <el-week-8>` | :ref:`Expense Tracker Week 8 <et-week-8>`.
+
 In Class
 --------
 
+Activity: Standup
+^^^^^^^^^^^^^^^^^
+
+Before we get started take a minute to have your standup meeting.
+
+Regroup: Intro to Continued Education Activity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For our final class today we want to talk about continued education. You
+must continue learning on your own after this class, and you must
+continue learning on the job. We will talk about how to do this.
+
+Activity: Continued Education
```

Comment:
this doesn't read to me as production ready for students to read. is there a decision yet to be made on what activity will be planned?

Reply:
@chrisbay - just this to clarify then its ready to go